### PR TITLE
Push Wasm extensions as OCI image at postsubmit.

### PR DESCRIPTION
**Pull request by bianpengyuan**
For wasm image pull integration test.

### CODE REVIEW

File: scripts/release-binary.sh [hunk 1]
```diff
@@ -211,6 +211,14 @@ trap 'rm -rf ${TMP_WASM}' EXIT
 make build_wasm
 if [ -n "${DST}" ]; then
   for extension in "${extensions[@]}"; do
+    if [ "${PUSH_DOCKER_IMAGE}" -eq 1 ]; then
```
Comment (zirain) [inline scripts/release-binary.sh:214]: can we use this image when `telemetry.metadataExchange.wasmEnabled: true`

Comment (zirain) [reply scripts/release-binary.sh:214]: BTW, do we have any data about the performance between `wasm metadataExchange` and `null metadataExchange`?

Comment (bianpengyuan) [reply scripts/release-binary.sh:214]: We used to measure it a long time ago. It is roughly 10% ~ 20% more expensive both cpu and latency. Although I heard it has been improved, so a new measurement probably would be needed.

Comment (zirain) [reply scripts/release-binary.sh:214]: once https://perf.dashboard.istio.io/ is fixed, we can setup a measurement daily.

**Review by kyessenov — APPROVED** [review]

**Comment by otterley** [comment]
@bianpengyuan Unfortunately this broke the build on the arm64 processor architecture; see https://github.com/istio/istio/issues/37913

**Pull request merged**